File: Selenium/Project/CRM_Activity1.py
# Goal: Read the title of the website and verify the text

# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

    # Start the Driver
with webdriver.Firefox() as driver:

    # Navigate to the URL
    driver.get("http://alchemy.hguy.co/crm")
    driver.maximize_window()

    # Get  the title of the page and verify it
    if(driver.title=="SuiteCRM"):
        print ("The Page title is : ",driver.title)
    else:
         print ("Wrong Website ")

    #close the browser
    driver.quit()

File: Selenium/Project/CRM_Activity1.java
package Project;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class CRM_Activity1 {
	WebDriver driver;

	@BeforeClass
	public void setUp() {
		// Initialize Webdriver
		driver = new FirefoxDriver();
		// Open the page
		driver.get("http://alchemy.hguy.co/crm");
		driver.manage().window().maximize();
	}

	@Test(priority = 1)
	public void pageTitleTest() {
		// Get the page title and verify it
		Assert.assertEquals(driver.getTitle(), "SuiteCRM", "Wrong Website");
		
	}

	@AfterClass
	public void browserClose() {
		// close the browser
		driver.quit();

	}
}
